vm/src/hint_processor/builtin_hint_processor/bootloader_hints.rs

```diff
@@ -272,4 +349,108 @@ mod tests {
             other => panic!("Expected an output builtin, found {:?}", other),
         }
     }
+
+    #[rstest]
+    fn test_load_bootloader_config(bootloader_input: BootloaderInput) {
+        let config = bootloader_input.bootloader_config.clone();
+
+        let mut vm = vm!();
+        // TODO: it is unclear why we need to create 2 segments and set fp = 2.
```

Comment:
Based on our conversation earlier, I think you figured this out, right?

Reply:
Half of it, yes.